presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestingArrowServer.java

```diff
@@ -222,7 +223,7 @@ else if (selectStatement != null) {
     @Override
     public Runnable acceptPut(CallContext callContext, FlightStream flightStream, StreamListener<PutResult> streamListener)
     {
-        return null;
+        throw new UnsupportedOperationException("This operation is not supported");
     }
```

Comment:
`listActions` should also throw unsupported operation exception

Reply:
done